clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -201,7 +201,7 @@ private void handleCatalogCommand() {
     String outputFormat = line.getOptionValue(GravitinoOptions.OUTPUT);
 
     if (CommandActions.LIST.equals(command)) {
-      newListCatalogs(url, ignore, metalake).handle();
+      newListCatalogs(url, ignore, metalake, outputFormat).handle();
```

Comment:
I'd suggest we come up with a ListOptions wrapper for this case.
In the foreseeable future, you may want to add filter, sorting, field selector, paging options. The list of parameters to pass would grow and you have to change all these places again and again. This process is tedious and error prone.

Reply:
I think we can pass the CommondOption value in the function params.
Through `CommondOption` we can get any option values.
@tengqm  @justinmclean What do you think?

Reply:
I'd suggest we don't overly generalize the options.
For example, you may want to model `ListOptions`, `DeleteOptions`, `UpdateOptions`.
The `outputFormat`, or `sortBy` option can then be encapsulated into `ListOptions`,
while an option like `force` can be encapsulated into `DeleteOptions`.

Reply:
I would not pass the CommondOption in the function parameters, so the command needs to know about the CLI library we use, and it makes it hard to know what options a command uses. Currently, we only have one list option and one delete option. I think we could change to a ListOptions or similar when adding extra options.